taxi/models.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+
+class Driver(AbstractUser):
```

Comment:
The `Driver` model is extending `AbstractUser`, which is a good approach when you need to add extra fields to the User model. However, according to the checklist, you should use `AUTH_USER_MODEL` instead of `AbstractUser`.